.env_vars.sample

```diff
@@ -32,6 +32,7 @@ export LIGHTSTEP_ACCESS_TOKEN=
 
 ## Sentry
 export NEXT_PUBLIC_SENTRY_DSN=https://6baf0213093a428d9e9ec7ae0979868c@o287463.ingest.sentry.io/5387583
+export SENTRY_ENVIRONMENT=
```

Comment:
```suggestion
export SENTRY_ENVIRONMENT= development
```

no?

Reply:
Now that I think about it, the line
```ts
enabled: process.env.NODE_ENV === "production",
```
should prevent any report in dev mode. That's why I did not add the variable in our local `env_vars`.